src/Control/Eff/QuickStart.hs

```diff
@@ -0,0 +1,226 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# OPTIONS_GHC -Wno-simplifiable-class-constraints #-}
```

Comment:
@power-fungus this was only introduced in GHC 8.2 onwards it seems like (based on CI errors) you may need to conditionally enable it